reference/3.0/PSDiagnostics/PSDiagnostics.md

```diff
@@ -1,18 +1,26 @@
 ---
-Module Name: PSDiagnostics
-Module Guid: c61d6278-02a3-4618-ae37-a524d40a7f44
-Download Help Link: {{Please enter FwLink manually}}
-Help Version: {{Please enter version of help manually (X.X.X.X) format}}
-Locale: en-US
+ms.date:  11/27/2018
+schema:  2.0.0
+locale:  en-US
+keywords:  powershell,cmdlet
+Help Version:  3.0.0.1
+Download Help Link:  https://go.microsoft.com/fwlink/?linkid=855965
+Module Guid:  c61d6278-02a3-4618-ae37-a524d40a7f44
+Module Name:  PSDiagnostics
 ---
 
 # PSDiagnostics Module
+
 ## Description
-{{Manually Enter Description Here}}
+
+The Windows PowerShell Diagnostics Module contains a set of cmdlets that enables the use of ETW
+tracing in PowerShell on Windows.
 
 ## PSDiagnostics Cmdlets
+
 ### [Disable-PSTrace](Disable-PSTrace.md)
-{{Manually Enter Disable-PSTrace Description Here}}
+
+Disables the Microsoft-Windows-PowerShell event provider logs.
 
 ### [Disable-PSWSManCombinedTrace](Disable-PSWSManCombinedTrace.md)
 {{Manually Enter Disable-PSWSManCombinedTrace Description Here}}
```

Comment:
Were these being cleaned up in this pass (I see others being filled in), or will another PR handle?

Reply:
Those are not being fixed in this PR. The issue was for PSv6. I updated the files affected by PSv6 and copied those to the other versions. I will work on the remaining files in a separate PR.